app/templates/help.hbs

```diff
@@ -9,26 +9,63 @@
     <div class="subheading align-center" data-test-help-page-navigation-links>
       View our most read
       <a href={{docsUrl}} target="_blank" ref="noopener noreferrer">documentation</a>,
-      popular topics in our
-      <a href={{communityUrl}} target="_blank" ref="noopener noreferrer">Community forum</a>,
-      or
-      <a class="pointer" onclick={{action 'setAnchor' 'form'}}>talk with us!</a>
+      {{#if isPro}}
+        popular topics in our
+        <a href={{communityUrl}} target="_blank" ref="noopener noreferrer">Community forum</a>,
+        or
+        <a class="pointer" onclick={{action 'setAnchor' 'form'}}>talk with us!</a>
+      {{else}}
+        or talk with us in our
+        <a href={{communityUrl}} target="_blank" ref="noopener noreferrer">Community forum</a>!
+      {{/if}}
     </div>
 
     <div class="status align-center" data-test-help-page-status>
       {{travis-status colorizeText=true }}
     </div>
   {{/layout.section}}
 
-  {{#layout.section class="help-page__support" hasBackground=(not isLoggedIn) data-test-help-page-support-section=true }}
-    {{#if toForm}}{{scroll-here}}{{/if}}
+  {{#layout.section class="help-page__resources" hasBackground=true data-test-help-page-resources-section=true}}
+    {{#if toDocs}}{{scroll-here}}{{/if}}
+    <div class="help-duo">
+      <div class="help-duo-image" data-test-help-page-resources-image>{{svg-jar 'docs'}}</div>
+      <div class="help-duo-text">
+        <h1 class="header-main" data-test-help-page-resources-header>Handy Resources</h1>
```

Comment:
This is entirely optional, but I would suggest to put content to a separate line in such cases, to make it stand out, for better readability. Usually when devs scan the code to find a specific element, they look for the text that is displayed on in UI. Putting it to a separate line would make it easier to distinguish:
```
<h1 class="header-main" data-test-help-page-resources-header>
  Handy Resources
</h1>
```

Reply:
Ooh yeah, that makes a lot of sense and does make it a lot easier to see what section you're working in. Change made, and I'll try to get in the habit of doing this :smile: